modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -513,26 +513,25 @@ label:new[Introduced in 5.26], the `FileSeedProvider`  supports:
 
 The `URLConnectionSeedProvider` supports the following:
 
-** `file:` label:deprecated[Deprecated in 5.26]
```

Comment:
Does it mean that support for `file` is removed? If so, in which Neo4j and Cypher versions?

Reply:
From Neo4j 2025.01 `file` is supported only with the `FileSeedProvider`. This is true for both Cypher 5 and Cypher 25.